CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.2](https://github.com/go-gitea/gitea/releases/tag/v1.10.2) - 2020-01-02
+* BUGFIXES
+  * [API] Allow only specific Colums to be updated on Issue (#9539) (#9580)
+  * Add ErrReactionAlreadyExist error (#9550) (#9564)
+  * Fix bug when migrate from API (#8631) (#9563)
+  * Use default avatar for ghost user (fix 500 error) (#9536) (#9537)
+  * Fix repository issues pagination bug when there are more than one label filter (#9512) (#9528)
+  * Fix deleted branch isn't removed when push the branch again (#9516) (#9524)
+  * Fix missing repository status when migrating repository via API (#9511)
+  * Trigger webhook when deleting a branch after merging a PR (#9510)
+  * [API] Fix paging on /repos/{owner}/{repo}/git/trees/{sha} endpoint (#9482)
```

Comment:
Please remove `[API]`

Reply:
@techknowlogick instead of [API] can i add  ... via API at the end?

Reply:
maybe add `API` in front of `endpoint`?